### ISSUE TITLE
关于test.py的一点问题

### ISSUE THREAD

**Issue by David9591** [zhaoyuzhi/Contexture-Encoder#1]
亲爱的作者：
         您好！
        首先感谢您分享这份代码，但是我在训练自己的数据(celeba)及并做测试时遇到一个问题：在训练的过程中，网络的输入是一个3通道的图像{masked_img=img*(1-mask)}，但是在测试的过程中，model的输入却变成4通道的图像{maskimg=torch.cat((maskimg,mask),1).cuda()}，会出现报错：`RuntimeError:Given groups=1,weight of size 64 3 4 4, expected input[1,4,258,258] to have 3 channels, but got 4 channels instead`，请问我是哪里出了问题呢？

**Comment by zhaoyuzhi** [comment]
Hi I think it is better to provide a mask channel accompanied with RGB image that serves as the 4-th channel.

**Comment by zhaoyuzhi** [comment]
Hi, I just update the test.py code. Could you please have a try for it? I think the original code is not appropriate for this model, not your problem. And you can also modify the code based on it.

**Comment by David9591** [comment]
Dear author, thanks for your quick reply！However,there still some problems in the new code.  Because nn.BatchNorm layer is used in the network model,so we can not set batch_size=1. If we do this,the program will report an error: `Error: Expected more than 1 value per channel when training.` I haven't found a solution yet,but I would like to express my heartfelt thanks to you for your work!



> Hi, I just update the test.py code. Could you please have a try for it? I think the original code is not appropriate for this model, not your problem. And you can also modify the code based on it.

**Comment by zhaoyuzhi** [comment]
Could you please update the file based on it? Or you may try model.eval()
